extensions/amp-timeago/1.0/component.js

```diff
@@ -39,36 +40,31 @@ export function BentoTimeago({
 
   const date = getDate(datetime);
 
-  useEffect(() => {
-    const node = ref.current;
-    const win = node && toWin(node.ownerDocument.defaultView);
-    if (!win) {
-      return undefined;
+  const ioCallback = ({isIntersecting}) => {
```

Comment:
This `ioCallback` is allocated every time the Component renders, and passed to `useIntersectionObserver`. `useIntersectionObserver` uses this as a dependency in its `useCallback`, meaning we're going to thrash the ref every time this renders.
```suggestion
  const ioCallback = useCallback(({isIntersecting}) => {
```

Reply:
Good point. And thanks for the explanation. Done.